locales/bn/json.json

```diff
@@ -212,15 +212,15 @@
     "Edit Attached": "সংযুক্ত সম্পাদনা করুন",
     "Edit Profile": "প্রোফাইল সম্পাদনা",
     "Editor": "সম্পাদক",
-    "Editor users have the ability to read, create, and update.": "সম্পাদক ব্যবহারকারীদের পড়তে তৈরি, এবং আপডেট করার ক্ষমতা রাখে ।",
+    "Editor users have the ability to read, create, and update.": "সম্পাদক ব্যবহারকারীরা পড়ার, তৈরির, এবং আপডেট করার ক্ষমতা রাখে।",
     "Egypt": "মিশর",
     "El Salvador": "এল সালভাদর",
     "Email": "ই-মেইল",
     "Email Address": "ই-মেইল ঠিকানা",
     "Email Addresses": "ই-মেইল ঠিকানা সমূহ",
     "Email Password Reset Link": "ইমেল পাসওয়ার্ড রিসেট লিংক",
     "Enable": "সক্রিয় করুন",
-    "Ensure your account is using a long, random password to stay secure.": "আপনার অ্যাকাউন্ট নিরাপদ থাকার একটি দীর্ঘ, এলোমেলো পাসওয়ার্ড ব্যবহার করা হয় তা নিশ্চিত করুন ।",
+    "Ensure your account is using a long, random password to stay secure.": "নিরাপদ থাকতে, আপনার অ্যাাকাউন্টে দীর্ঘ, এলোমেলো পাসওয়ার্ডের ব্যাবহার নিশ্চিত করুন।",
```

Comment:
Here please change the word "থাকতে" to "রাখতে"

Reply:
done